swift/apple/FirezoneNetworkExtension/SystemConfigurationResolvers.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  SystemConfiguration.swift
+//  FirezoneNetworkExtensionmacOS
+//
+//  Created by Jamil Bou Kheir on 2/26/24.
+//
+
+import FirezoneKit
+import Foundation
+import SystemConfiguration
+
+class SystemConfigurationResolvers {
+  private let logger: AppLogger
+  private var dynamicStore: SCDynamicStore?
```

Comment:
I looked this up and the Apple docs just said it's a handle to "the dynamic store of a running system". It didn't say what a dynamic store is.

I'm guessing it's a key-value store that doesn't survive reboots?